cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,77 @@
 /// <reference types='cypress' />
 
+
 describe('Student Registration page', () => {
   before(() => {
 
+    cy.visit('https://demoqa.com/automation-practice-form');
+    cy.viewport(1920, 1080);
   });
 
-  it('', () => {
+  
+  it('Automation registration form', () => {
+    const firstName = 'orest';
+    const lastName = 'NOob';
+    const email = 'trojanhorse3@gmail.com';
+    const phoneNumber = '0677033886';
+    const subject = 'English';
+    const adress = 'Ukraine, Moscov';
+    cy.get('#firstName')
+      .type(firstName);
+    cy.get('#lastName')
+      .type(lastName);
+    cy.get('#userEmail')
+      .type(email);
+    cy.get('[for="gender-radio-1"]')
+      .click();
+    cy.get('#userNumber')
+      .type(phoneNumber);
+    cy.get('#dateOfBirthInput')
+      .click();
+    cy.get('.react-datepicker__month-select')
+      .select('July');
+    cy.get('.react-datepicker__year-select')
+      .select('2008');
+    cy.get('.react-datepicker__day--014')
+      .click();
+    cy.get('.subjects-auto-complete__value-container')
+      .type(subject + `{Enter}`);
+    cy.get('[for="hobbies-checkbox-1"]')
+      .click();
+    cy.get('[placeholder="Current Address"]')
+      .type(adress);
+    cy.get('#state')
+      .click();
+    cy.get('#react-select-3-option-0')
+      .click();
```

Comment:
I think it would be better to do this
"cy.get('#state')
      .type('{downarrow}{enter}');"